src/main/java/worldofblocks/game/Game.java

```diff
@@ -35,7 +35,7 @@ private void init() {
     this.window = new Window(windowWidth, windowHeight, fullscreen);
 
     // hide cursor but enable mouse grab.
-    glfwSetInputMode(window.getId(), GLFW_CURSOR, GLFW_CURSOR_DISABLED);
+//    glfwSetInputMode(window.getId(), GLFW_CURSOR, GLFW_CURSOR_DISABLED);
```

Comment:
don't forget to re-enable this before merging :cactus: 

It would be handy to disable this when running a debug build.